### ISSUE TITLE
Why the loss is very big

### ISSUE THREAD

**Issue by Dengxianshuai** [meetshah1995/pytorch-semseg#212]

**Comment by Dengxianshuai** [comment]
When I training icnet, why the loss is very big.
![1557914556(1)](https://user-images.githubusercontent.com/30050929/57767346-b251ec80-773b-11e9-9582-aca195fc304a.png)

**Comment by jamesbondzhou** [comment]
did u fix the problem yet？

**Comment by flyking1994** [comment]
> did u fix the problem yet？
I met the same problem, did u fix it?

**Comment by flyking1994** [comment]
> > did u fix the problem yet？
> > I met the same problem, did u fix it?

    loss:
        name: 'cross_entropy'
        size_average: True

change the size_average into True.
